parquet-column/src/main/java/org/apache/parquet/column/statistics/SizeStatistics.java

```diff
@@ -67,8 +67,16 @@ public static class Builder {
     private Builder(PrimitiveType type, int maxRepetitionLevel, int maxDefinitionLevel) {
       this.type = type;
       this.unencodedByteArrayDataBytes = 0L;
-      repetitionLevelHistogram = new long[maxRepetitionLevel + 1];
-      definitionLevelHistogram = new long[maxDefinitionLevel + 1];
+      if (maxRepetitionLevel > 0) {
+        repetitionLevelHistogram = new long[maxRepetitionLevel + 1];
+      } else {
+        repetitionLevelHistogram = new long[0]; // omitted
+      }
+      if (maxDefinitionLevel > 1) {
```

Comment:
I'm curious why you've chosen 1 here, when other implementations use 0. Is it so parquet-java will implement the spec exactly or are you thinking to minimize metadata size? Not a criticism, just genuinely curious.